index.html

```diff
@@ -380,12 +380,12 @@
         <!-- Club Section Ends Here-->
 
         <!-- FAQ Section Starts Here -->
-        <section  id="faq" class="container" style="padding-left:0px">
+        <section  id="faq" class="mt-0" style="padding-left:0rem">
```

Comment:
Replace in-line CSS with Bootstrap utilities too please. There are multiple cases of this throughout the FAQ Section, would fix all of them.

Reply:
I have remove inline css and add bootstrap utilities in it